pkg/proxy/winkernel/proxier.go

```diff
@@ -278,6 +278,24 @@ func newServiceInfo(svcPortName proxy.ServicePortName, port *v1.ServicePort, ser
 	return info
 }
 
+func (network hnsNetworkInfo) findProviderAddress(ip string) string {
```

Comment:
can you name this as , findRemoteSubnetProviderAddress

Reply:
renamed the method